counterparty-core/counterpartycore/test/compare_hashes_test.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+import pytest
+import requests
+
+from counterpartycore.test import (
+    conftest,  # noqa: F401
+)
+
+LOCAL_API_URL = "http://api:api@localhost:4000"
+
+# [server_url, api_version]
+CHECK_SERVERS = [
+    ["http://rpc:rpc@api1.counterparty.io:4000", "v1"],  # v9.61.3
```

Comment:
hmm I think we should record which version is supposed to be running on each server and have this test verify that before passing

Reply:
yep good idea! done!
(so api1 is running v9.61.1)